src/app/components/mno-welcome-wizard/mno-welcome-wizard.less

```diff
@@ -0,0 +1,49 @@
+#welcome{
+  min-height: 85%;
+  max-height: 85%;
```

Comment:
@alexnoox
I have read that using max-height and min-height, I gain more control. In fact, when I switch this to height:85%, the overflow hidden scrollable stops working and therefore the elements come out the box.